### ISSUE TITLE
Falha em utilização do sistema devido ao tamanho do cookie 'portalservicos.senatran.serpro'

### ISSUE THREAD

**Issue by GuilhermeBley** [servicosgovbr/manual-script-chatbot-login-unico#40]
## Falha ao acessar serviço Senatran:

![image](https://user-images.githubusercontent.com/69880922/191776282-89386c3f-8be5-4c0e-b0c9-0693174af75c.png)

Problema ocorreu após adicionar empresas vinculadas ao CPF.

Provavelmente seja por conta do tamanho do cookie que pode estar interferindo no Token JWT, já que cada nova empresa vinculada aumenta o seu tamanho, estando já bem próximo ao máximo:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Bley/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Bley/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

Name | Value | Domain | Path | Expires | Size
-- | -- | -- | -- | -- | --
SESSION-ID | ey... | portalservicos.senatran.serpro.gov.br | / | Session | 3591

</body>

</html>

</br>
</br>
Devido à isso, não é possível ter acesso ao sistema.

**Comment by caduvieira** [comment]
A escolha entre session/cookie/local storage depende do site. Creio que nesse repositório não tem ninguém responsável pelo SENATRAN.

@higogomes o que acha?

**Comment by GuilhermeBley** [comment]
Sim, quanto à isso não há problema, gostaria de verificar se realmente seria isso o problema, porque até agora não encontrei outra justificativa da falta de acesso que estamos tendo aqui em nossa empresa.

E caso vocês conheçam algum canal mais ideal para essa pergunta, me passem por favor.
